### PR TITLE
Issue #119

### PR DESCRIPTION
**Pull request by arocketman**
So, new Reddit users can't comment too fast. There's a limit of 10 minutes for certain subreddits.
The comment() method returned true even tho it was not effectively posting.
I described the problem here #119 .

This pull requests aims to fix the issue. 

There are two commits because I had to fix the indentation in the second one.

### CODE REVIEW

**Comment by sfat** [comment]
@arocketman thanks for the contribution. Would you mind if you to update this https://github.com/karan/jReddit/blob/master/src/test/java/com/github/jreddit/action/SubmitActionsTest.java to include this case, please?

**Comment by arocketman** [comment]
I included the case. Please double check it because I didn't work with mockito before.

File: src/test/java/com/github/jreddit/action/SubmitActionsTest.java [hunk 1]
```diff
@@ -116,6 +121,28 @@ public void testCommentUserRequired() {
...
+                + "&uh=" + user.getModhash(),
+                ApiEndpointUtils.COMMENT, user.getCookie()))
+                .thenReturn(responseTenMinutesLimit);
+        // Save System.err.println output to variable
+        ByteArrayOutputStream errorOutput = new ByteArrayOutputStream();
+        System.setErr(new PrintStream(errorOutput));
+
+        // Attempt to comment
+        boolean res = submitAction.comment("fullname", "text");
+        assertEquals(errorOutput.toString().contains("User submission failed: You need to wait before posting again"), true);
```
Comment (sfat) [inline src/test/java/com/github/jreddit/action/SubmitActionsTest.java]: assertTrue() can be replaced here

Comment (arocketman) [inline src/test/java/com/github/jreddit/action/SubmitActionsTest.java]: You mean at line 143 ? 

I just followed the style of testCommentUserRequired() which uses assertEquals

Comment (sfat) [inline src/test/java/com/github/jreddit/action/SubmitActionsTest.java]: yeah, that's why the comment was put on line 143 :). I didn't notice the other one. It's not a big deal, just wanted to increase the readability of the test. If you don't mind, you want to change the other one as well? I will be grateful forever :+1:

Comment (arocketman) [inline src/test/java/com/github/jreddit/action/SubmitActionsTest.java]: Sure thing. Gimme a sec.

**Comment by sfat** [comment]
@arocketman thanks a lot!

**Comment by sfat** [comment]
forgot to merge, whoops :no_mouth:

**Pull request merged**